packages/cli/README.md

```diff
@@ -23,80 +23,43 @@ yarn add @pantheon-systems/pcc-cli --global
 ## Usage
 
 ```bash
-$ # Show Help
-$ pcc init --help
-pcc init <project_directory> [options]
+# Login the user
+$ pcc login
 
-Sets up project with required files.
-
-Positionals:
-  <project_directory>  The project directory in which setup should be done.
-                                                                        [string]
-
-Options:
-  --version   Show version number                                      [boolean]
-  --help      Show help                                                [boolean]
-  --template  Template from which files should be copied.
-                               [string] [required] [choices: "nextjs", "gatsby", "vue"]
-
-$ # Create project with nextjs template (this is  default)
+# Create project with nextjs template
 $ pcc init new_proj
-✔ Fetched starter kit!
-✔ Completed setting up project!
-
-To get started please run:
-   cd new_proj
-
-   # Set your site id and API key
-   vim .env.local
 
-   # Run the site
-   yarn dev
-
-$ # Create project with gatsby template
+# Create project with gatsby template
 $ pcc init new_proj --template=gatsby
-✔ Fetched starter kit!
-✔ Completed setting up project!
-
-To get started please run:
-   cd new_proj
-
-   # Set your site id and API key
-   vim .env.development
-
-   # Run the site
-   yarn start
 
-$ # Retrieve component schema from your playground site
-$ pcc site components --url https://live-collabcms-fe-demo.appa.pantheon.site --apiPath /api/YOUR_SITE_ID/pantheoncloud/component_schema
+# Create project using pnpm package manager
+$ pcc init new_proj --use-pnpm
 
-$ # Retrieve component schema from your site (replace www.example.com)
-$ pcc site components --url https://www.example.com
+# Create Typescript project and setup ESLint in it
+$ pcc init new_proj --ts --eslint
 
-```
+# Create project without installing dependencies
+$ pcc init new_proj --noInstall
 
-## Create site from Vue template
+# Create project and provide site ID to pre-populate .env file with
+$ pcc init new_proj --site-id 8YUsfuf1EhjLGhswM49q
```

Comment:
You need to provide some clearly fake id here. Expect any value you use in documentation to be used verbatim by some users. Maybe you can set it to something like 12345example6789